src/imgtools/cli/autopipeline.py

```
@@ -0,0 +1,245 @@
+    apply transformations, and save the results to a specified output directory.
+    """
+    # Parse ROI match map
+    roi_map = {}
+    
+    # Load from YAML file if provided
+    if roi_match_yaml:
+        try:
+            with open(roi_match_yaml, 'r') as f:
+                yaml_data = yaml.safe_load(f)
+                if not isinstance(yaml_data, dict):
+                    raise click.BadParameter(f"ROI match YAML must contain a dictionary")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)